bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdf_pyz.py

```diff
@@ -24,7 +22,7 @@ class FunctionJitter:
     Attributes
     ----------
     rdf: RDataFrame object on which the function's column arguments are in
-    col_names: The list of columns in rdf
+    col_names: The list of columns in rdf 
```

Comment:
Extra space (also in other lines below, please check).

Reply:
I have fixed the spaces. Hopefully. 
Not sure where the auto-formatting was triggered from. Something with the editor.